modules/nw-egress-router-http-proxy-mode.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_sdn/deploying-egress-router-http-redirection.adoc
+
+[id="nw-egress-router-http-proxy-mode_{context}"]
+= Deploying an egress router pod in HTTP proxy mode
+
+In _HTTP proxy mode_, an egress router pod runs as an HTTP proxy on port `8080`. This only works for clients connecting to HTTP or HTTPS-based services, but usually requires fewer changes to the client pods. Applications must be configured to use an HTTP proxy by setting an environment variable.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create an egress router pod.
+
+. Ensure other pods can find the IP address of the egress router pod by creating a service to point to the egress router pod, as in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: egress-1
+spec:
+  ports:
+  - name: http-proxy
+    port: 8080 <1>
+  type: ClusterIP
+  selector:
+    name: egress-1
+----
+<1> Ensure the `http` port is always set to `8080`.
+
+. Configure the client pod (not the egress proxy pod) to use the HTTP proxy by setting the `http_proxy` or `https_proxy` variables:
```

Comment:
Similar wording suggestions to avoid passive and make user action (set) clear:
> To configure the client pod (not the egress proxy pod) to use the HTTP proxy, set the `http_proxy` or `https_proxy` variables:

I think we're supposed to avoid parens but I also think these make sense here :slightly_smiling_face: